toolchain/check/import_cpp.cpp

```diff
@@ -427,6 +444,28 @@ static auto GetReturnType(Context& context, SemIR::LocId loc_id,
   return param_pattern_id;
 }
 
+// Returns parameter patterns block id, the return slot pattern id, and the
+// call parameters block id for the given function declaration.
+// Returns false if the function declaration has an unsupported parameter type.
+static auto GetFunctionParams(Context& context, SemIR::LocId loc_id,
+                              clang::FunctionDecl* clang_decl,
```

Comment:
I believe this can be const.
```suggestion
                              const clang::FunctionDecl* clang_decl,
```

Reply:
Done.